custom_components/spook/translations/de.json

```diff
@@ -0,0 +1,65 @@
+{
+  "config": {
+    "abort": {
+      "already_spooked": "Ja... nun... nein. Okay, danke, tschüss!"
+    },
+    "step": {
+      "user": {
+        "description": "Hast du die Readme gelesen? Hast du die Lizenz gelesen? Bist du dir wirklich sicher?"
+      }
+    }
+  },
+  "issues": {
+    "user_issue": {
+      "title": "{title}",
+      "fix_flow": {
+        "step": {
+          "confirm": {
+            "title": "{title}",
+            "description": "{description}"
+          }
+        }
+      }
+    },
```

Comment:
This will be automatically picked up (fallback), as there are no translations in this part.

```suggestion
```